lib/pgrel/active_record/relation.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module ActiveRecord
+  class Relation
+    def update_store(store_name)
+      raise ArgumentError, "Empty store name to update" if store_name.blank?
+      type = type_for_attribute(store_name.to_s).type
+      raise TypeConflictError, "Column type mismatch" if %i(hstore jsonb).exclude?(type)
```

Comment:
Let's add `type` to the error message, e.g.: `"Column type is not a known store: #{type}"`